pages/deiab-highlight.html

```diff
@@ -11,10 +11,10 @@
 path-order: 2
 path-total: 4
 prx-path: partner
-quote: "<p>&ldquo;Equity work, at its heart, is a series of choices every day — how we show up, connect with each other, and speak with each other. The conversations we elevate and voices we engage with. We make choices by the minute. My hope is to provide more knowledge around how we make decisions and choose our adventure.&rdquo;</p>"
-quote-citation: 'Dr. Byron Green-Calisch, Vice President of IDEA'
-quote-image: img/prx-cookout-byron.jpg
-quote-image-alt: "A Black man speaks in front of a microphone"
+donor-quote: "<p>&ldquo;Equity work, at its heart, is a series of choices every day — how we show up, connect with each other, and speak with each other. The conversations we elevate and voices we engage with. We make choices by the minute. My hope is to provide more knowledge around how we make decisions and choose our adventure.&rdquo;</p>"
+donor-quote-citation: 'Dr. Byron Green-Calisch, Vice President of IDEA'
+donor-quote-image: img/prx-cookout-byron.jpg
+donor-quote-image-alt: "A Black man speaks in front of a microphone"
```

Comment:
Donor quotes don't have the arrow pointing. Since this quote is the last thing we don't want an arrow to nowhere.

Reply:
Could you have used a `&:last-child` selector to hide the arrows with CSS?

Reply:
If it happened in more than two sections, I likely would have.